crates/ibc/src/core/ics03_connection/delay.rs

```diff
@@ -0,0 +1,49 @@
+use crate::core::{ics02_client::height::Height, ContextError, ValidationContext};
+
+use super::{connection::ConnectionEnd, error::ConnectionError};
+
+pub fn verify_conn_delay_passed<Ctx>(
+    ctx: &Ctx,
+    height: Height,
```

Comment:
I'd rename this to `packet_commitment_proof_height` (or simply `packet_proof_height`?), or something more descriptive?

Reply:
[010f87e](https://github.com/cosmos/ibc-rs/pull/498/commits/010f87ee7629d291dcbb350c5d24ed87cd5f468a)